test/app-tap/datetime.test.lua

```diff
@@ -1108,6 +1201,90 @@ test:test("Matrix of allowed time and interval subtractions", function(test)
     test:is(catchsub_status(min_date, new_ival{week = 100}), false, "min - 100wk")
 end)
 
+test:test("Parse iso date - valid strings", function(test)
```

Comment:
Could you replace 'iso' with 'iso8601' here and for other tests?

Reply:
Done